runtime-display/jh7110-pvr-blobs/autobuild/overrides/usr/lib/dracut/dracut.conf.d/50-jh7110-pvr.conf

```diff
@@ -0,0 +1 @@
+omit_drivers+=" pvrsrvkm "
```

Comment:
Why?

Reply:
When the kernel module `pvrsrvkm` exists in the initramfs but does not include the corresponding firmware, the GPU firmware loading will fail. For devices like VisionFive2, initramfs should be optional rather than necessary, so I choose to ignore this kernel module in initramfs and switch to dynamic loading instead.

Reply:
Please include this in the file as a comment.